schedule/yast/maintenance/autoupgrade_sle15sp4-1_scc_we_sdk_basesys_wsm_srv_desk_lgm_contm_pcm_def_full.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: autoupgrade_sle15sp4-1_scc_we_sdk_basesys_wsm_srv_desk_lgm_contm_pcm_def_full
+description: >
+  Performs a migration between installed system with unreleased maintenance updates installed, and next release, eg sle15 sp3>sp4
+schedule:
+  - autoyast/prepare_profile
+  - migration/version_switch_origin_system
+  - boot/boot_to_desktop
+  - console/setup_libyui_running_system
```

Comment:
you don't need this setup (as no module which enable libyui rest api is using).
same for test data below.

Reply:
Updated, thanks.